cfme/fixtures/cli.py

```diff
@@ -1,17 +1,26 @@
-import pytest
-
 from collections import namedtuple
 from contextlib import contextmanager
+
+import fauxfactory
+import pytest
 from six import iteritems
 
 import cfme.utils.auth as authutil
-from cfme.test_framework.sprout.client import SproutClient
-from cfme.utils.conf import cfme_data, credentials, auth_data
+from cfme.cloud.provider.ec2 import EC2Provider
+from cfme.configure.configuration.region_settings import RedHatUpdates
+from cfme.infrastructure.provider.virtualcenter import VMwareProvider
+from cfme.test_framework.sprout.client import SproutClient, SproutException
+from cfme.utils import conf
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.conf import credentials, auth_data, cfme_data
 from cfme.utils.log import logger
+from cfme.utils.providers import list_providers_by_class
+from cfme.utils.version import Version
```

Comment:
Please remove the line below and use
from cfme.utils.version import get_stream , Version